src/transport_ofi.c

```diff
@@ -1199,10 +1223,11 @@ int query_for_fabric(struct fabric_info *info)
     shmem_transport_ofi_mr_rma_event = (info->p_info->domain_attr->mr_mode & FI_MR_RMA_EVENT) != 0;
 #endif
 
-    DEBUG_MSG("OFI provider: %s, fabric: %s, domain: %s\n"
+    DEBUG_MSG("OFI provider: %s, fabric: %s, domain: %s, mr_mode: 0x%x\n"
               RAISE_PE_PREFIX "max_inject: %zu, max_msg: %zu, stx: %s, stx_max: %ld\n",
               info->p_info->fabric_attr->prov_name,
               info->p_info->fabric_attr->name, info->p_info->domain_attr->name,
+              info->p_info->domain_attr->mr_mode,
```

Comment:
We could, although it will be a constant that is determined by `--enable-ofi-mr=...` and can be figured out by looking at the configuration args already printed in the debug info.

Reply:
I didn't test with GNI or PSM2. The change is minimal with scalable MR. You would need to also test with basic MR, and the provider may just revert back to scalable by clearing all the mr_mode bits.

Reply:
Yeah... Using SOS w/GNI requires MR_BASIC, right?  If the provider reverts to scalable, wouldn't we run into a problem?

Reply:
If the provider clears all of the mr_mode bits, then it is operating with scalable MR even if SOS isn't taking advantage of it. This is what the sockets provider does in the SOS basic MR configuration.